Fixed issues in Swedish braille tables

InsynScandinavia opened the pull request.

bertfrees reviewed (changes requested):
Thanks! Looks good, but I have a few comments:

- Can you explain the changes to the metadata. Some fields were changed, some were moved and some deleted.
- Some copyright lines where added. Was that intentional? The year of your own copyright should be 2022-2023, not 2023.
- I would be good if the tests would be executed in both directions, especially now that the direction metadata was changed to "both".

InsynScandinavia commented:
Hello,
Thanks for your comments.

  *   I made some changes to the metadata in error. The only real changes I intended was to change the direction to both, since this is how the tables have been used all the time.
  *   I’ve corrected the copyright lines.
  *   I’ve updated the sv.yaml file to just perform forward testing for some tests and both directions for the rest.

Thanks,
/David R.

egli commented:
We need to deal with the problem that this table might crash a liblouis that hasn't been built with ucs4

bertfrees commented on tests/braille-specs/sv.yaml:
```diff
@@ -77,13 +71,16 @@ table:
+  - # Non-contraction letters
+    - Nej s som i Sigurd.
+    - ⠠⠝⠑⠚ ⠒⠎ ⠎ ⠊ ⠠⠎⠊⠛⠥⠗⠙⠄
+flags: {testmode: forward}
```
An alternative for running some tests only in forward direction is to use the [enhanced  `xfail` flag which now allows specifying the direction](https://github.com/liblouis/liblouis/issues/1223). I can have a look at this.

bertfrees replied:
Done

bertfrees commented:
@InsynScandinavia Thanks for the swift response. Unfortunately we decided to move the PR to the next milestone because the test fails and we don't have enough time left to figure out what is wrong.

bertfrees commented:
@InsynScandinavia We would also like to say something about this update in the NEWS file. Can we say something more informative than "Improvements to the Swedish table"?

bertfrees commented:
> We need to deal with the problem that this table might crash a liblouis that hasn't been built with ucs4

@egli It's not related to UCS2. The UCS4 build would also have failed, but it was cancelled when the UCS2 build failed.

InsynScandinavia commented:
We can say:
Improved handling of numbers as well as fixed a couple of whole-word contraction errors.

InsynScandinavia commented:
What tests are failing?
